bridge-adapters/kinetic-bridgehub-adapter-ars/src/test/resources/bridge-config.yml

```diff
@@ -5,8 +5,8 @@ Bridge Configuration:
     Username: ""
     Password: ""
     Server: ""
-    Port: ""
-    Prognum: ""
+    Port: "3000"
```

Comment:
Set Port and Prognum back to empty strings, unless there was a good reason to change these.

Reply:
The placeholder values have been removed.